### PR TITLE
Add coupon code field to Promotion content type and fill it during sync.

### PR DESCRIPTION
**Pull request by miromichalicka**
This PR is addressing new feature request to be able to store coupon code in Drupal. We're adding new plain text field Coupon code(field_coupon_code) with unlimited cardinality to Promotion content type and we're filling this field during promotion sync.

### CODE REVIEW

File: modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml [hunk 1]
```diff
@@ -118,19 +119,21 @@ content:
...
+    type: string_textfield
+    region: content
   field_skus:
     weight: 20
     settings:
       size: 60
     third_party_settings: {  }
     type: sku_widget
     region: content
-  langcode:
```
Comment (sdelbosc) [inline modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml:128]: @miromichalicka why removing langcode?

Comment (miromichalicka) [reply modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml:128]: i ididn't remove it manually, this was exported from latest drupal

Comment (sdelbosc) [reply modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml:128]: I wonder if this is going to make the content type not translatable while it should be

Comment (miromichalicka) [reply modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml:128]: No, you just won't see language selector while editing

Comment (sdelbosc) [reply modules/promotion/config/install/core.entity_form_display.node.acm_promotion.default.yml:128]: pr looks fine to me then

**Review by csibar — APPROVED** [review]

**Pull request merged**